src/isUniqueArray.js

```diff
@@ -0,0 +1,15 @@
+const isUniqueArray = (array) => {
+  const result = [];
+
+  for (const number of array) {
+    if (!result.includes(number)) {
+      result.push(number);
+    } else {
+      return false;
+    }
```

Comment:
no need for return
```suggestion
    }
```

Reply:
If I delete this return, I`ll get this.
![image](https://github.com/mate-academy/node_bulls-and-cows/assets/133264109/7dc6e890-271c-4b84-802a-a9037bc1b83d)